test/electionguard/test_ballot.cpp

```diff
@@ -47,3 +54,37 @@ TEST_CASE("Plaintext Ballot Selection Is InValid")
         CHECK(subject->isValid(objectId) == false);
     }
 }
+
+TEST_CASE("Can serialize PlaintextBallot")
+{
+    // Arrange
+    auto metadata = ElectionGenerator::getFakeMetadata(TWO_MOD_Q());
+    auto plaintext = BallotGenerator::getFakeBallot(*metadata);
+    auto json = plaintext->toJson();
+    auto bson = plaintext->toBson();
+
+    // Act
+    auto fromJson = PlaintextBallot::fromJson(json);
+    auto fromBson = PlaintextBallot::fromBson(bson);
```

Comment:
Does this also not need to check msg pack because it's inside the compact one?

Reply:
cant hurt